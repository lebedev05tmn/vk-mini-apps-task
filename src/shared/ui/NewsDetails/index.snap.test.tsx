import { NewsDetails } from "./index";

const ReactTestRenderer = require("react-test-renderer");

describe("News Details render", () => {
  test("Should NewsDetails render", () => {
    const tree = ReactTestRenderer.create(
      <NewsDetails
        by={"author"}
        time={"12345"}
        title={"title"}
        url={"#"}
        descendants={1}
      />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
