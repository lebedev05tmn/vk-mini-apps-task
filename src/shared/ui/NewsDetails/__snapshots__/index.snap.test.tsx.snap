// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`News Details render Should NewsDetails render 1`] = `
<div
  className="vkuiCard"
>
  <span
    className="vkuiTypography vkuiTypography--normalize vkuiTitle--level-1"
  >
    Новость: 
    title
  </span>
  <span
    className="vkuiTypography vkuiParagraph"
  >
    Автор новости: 
    author
  </span>
  <span
    className="vkuiTypography vkuiParagraph"
  >
    Ссылка на новость:
     
    <a
      href="#"
      target="_blank"
    >
      #
    </a>
  </span>
  <span
    className="vkuiTypography vkuiParagraph"
  >
    Количество комментариев: 
    1
  </span>
</div>
`;
